### ISSUE TITLE
AttributeError: 'EntryPoints' object has no attribute 'get'

### ISSUE THREAD

**Issue by MehmedGIT** [hnesk/browse-ocrd#57]
What is going wrong here? Native installation or installation using pip ends up producing the same error. 

```bash

mm@MM-Notebook:~$ sudo docker run -it --rm -v /home/mm/Desktop/workflows/:/data -p 8085:8085 -p 8080:8080 hnesk/ocrd_browser
+ python serve.py -p 8080 -P 8085 -d /data
+ broadwayd :5
Listening on /root/.cache/broadway6.socket
/data/ocrd-ws-serial-cleaned/mets.xml
/data/ocrd-ws-serial/mets.xml
/data/ocrd-ws-pagewise/mets.xml
/data/ocrd-ws-pagewise-cleaned/mets.xml
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:05] "GET /favicon.ico HTTP/1.1" 403 -
172.17.0.1 - - [30/Nov/2022 23:02:10] "GET /browse/ocrd-ws-serial-cleaned/mets.xml HTTP/1.1" 303 -
Traceback (most recent call last):
  File "/usr/local/bin/browse-ocrd", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/ocrd_browser/main.py", line 63, in main
    app = OcrdBrowserApplication()
  File "/usr/local/lib/python3.7/site-packages/ocrd_browser/application.py", line 28, in __init__
    self.view_registry = ViewRegistry.create_from_entry_points()
  File "/usr/local/lib/python3.7/site-packages/ocrd_browser/view/registry.py", line 21, in create_from_entry_points
    for entry_point in entry_points().get('ocrd_browser_view', []):
AttributeError: 'EntryPoints' object has no attribute 'get'
```

**Comment by bertsky** [comment]
See https://github.com/hnesk/browse-ocrd/pull/56 and https://github.com/hnesk/browse-ocrd/pull/53

**Comment by hnesk** [comment]
Fixed by @bertsky in #53 and #56. Sorry for the delay.
